### ISSUE TITLE
problem when I use mysql

### ISSUE THREAD

**Issue by gabykode** [fguillot/kanboard#751]
Dear Frédéric,

I want to use your app with mysql. I configured all (http://kanboard.net/documentation/mysql-configuration) but I don't know why I have this message:

Unable to migrate database schema: 

\Schema\version_32 => SQLSTATE[42S21]: Column already exists: 1060 Duplicate column name 'date_started'

sorry, I am a new programmer

thank you

**Comment by fguillot** [comment]
Is it a new installation or an update of Kanboard? if it's a new installation try to drop and create again the database.

**Comment by fguillot** [comment]
No answer -> closed.

**Comment by c99ipnerd** [comment]
Hello Frédéric,

i would like to reopen the issue. I tried updating from 1.0.12 -> 1.0.14. I did also try an update with the git tag 1.0.13, but that showed the same message.

```
Unable to migrate database schema:

\Schema\version_48 => SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'files' already exists
```
Yep, it exists and it is not the only table it complaints on after I deleted the table. The table schema_version shows 47.

I had set the DEBUG to true in config.php, but it didn't create a log file.

Thanks in advance.

regards,
Alwin
